package.json

```diff
@@ -29,6 +29,7 @@
     "readme": "./bin/plugin/cli.js readme",
     "translations": "./bin/plugin/cli.js translations",
     "build": "wp-scripts build",
+    "build-plugin": "wp-scripts build --env build=plugin",
     "build-plugins": "./bin/plugin/cli.js build-plugins",
```

Comment:
This is confusing to me. What is the purpose of `build-plugin` and `build`? We already have `build-plugins` which is for the modules to be built into plugins (which can soon be removed). Do we really need `build-plugin` or can it be combined somehow? If not, let's use a different name that clarifies what this is for.